web/src/components/FormStateStart.js

```diff
@@ -1,37 +1,44 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import { Divider, Select } from 'rebass';
 import { Field, reduxForm } from 'redux-form';
 
 import { Input } from './Inputs';
+import SectionHeader from './SectionHeader';
+import SelectInput from './SelectInput';
+import { STATES } from '../util';
 
 const FormStateStart = ({ handleSubmit, pristine, reset, submitting }) => (
   <form onSubmit={handleSubmit}>
-    <Field name="firstName" type="text" component={Input} label="First Name" />
-    <Field name="lastName" type="text" component={Input} label="Last Name" />
+    <SectionHeader>Your info:</SectionHeader>
+    <Field name="name" type="text" component={Input} label="Name" />
+    <Field name="position" type="text" component={Input} label="Position" />
+    <Field name="email" type="email" component={Input} label="Email address" />
+    <Field
+      name="phoneNumber"
+      type="tel"
+      pattern="[0-9]{3}-[0-9]{3}-[0-9]{4}"
+      component={Input}
+      label="Phone number"
+    />
 
-    <div>
-      <label htmlFor="favoriteColor">Favorite Color</label>
+    <SectionHeader>State:</SectionHeader>
```

Comment:
Let's leave it for now, but I'll be interested to see what design has to say about this header after a little bit of testing.  Not every entity submitted APDs is a state.  😆